src/lib/Calculator.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+export default Calculator;
+
+function Calculator () {
```

Comment:
Any particular reason to use function constructor (without prototype chain) over `class `syntax?

Reply:
No particular reason - it's just how I'm used to writing constructors unless the prototype chain is needed or the class syntax is preferred, like within a Vue component. Any of these patterns would be totally fine to use.

Reply:
I see. Since you're not using any private variables within the Calculator function, I think it would be better to extract all methods to Calculator's prototype. This would be beneficial in case there might be more than one component or Calculator instance used (methods don't have to be recreated for every new instance).

Reply:
Agreed. Just updated lib/Calculator.js to use the class syntax. Thanks for the feedback!